packages/openneuro-components/src/form/RadioGroup.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react'
+import { Radio } from './Radio'
+
+import './forms.scss'
+
+export interface RadioGroupProps {
+  id: string
+  layout: string
+}
+
+export const RadioGroup: React.FC<RadioGroupProps> = ({
+  radioArr,
+  id,
+  layout,
+  ...props
+}) => {
+  const [active, setActive] = React.useState(0)
+
+  return (
+    <div className={'on-radio-wrapper' + ' ' + layout} {...props} id={id}>
+      {radioArr.map((item, index) => (
+        <Radio
+          name={item.name}
```

Comment:
Since all the radio buttons in a `RadioGroup` should always have the same name, I think the `name` can be a prop for `RadioGroup` and then the `radioArr` doesn't need a `name` property